ambient-light/idlharness.html

```diff
@@ -98,7 +98,7 @@
   idl_array.add_objects({
     AmbientLightSensor: ['new AmbientLightSensor();'],
     AmbientLightSensorReading: ['new AmbientLightSensorReading({ illuminance: 750 });'],
-    SensorReadingEvent: ['new SensorReadingEvent({ reading: new AmbientLightSensorReading({ illuminance: 750 }) });']
+    SensorReadingEvent: ['new SensorReadingEvent("", { reading: new AmbientLightSensorReading({ illuminance: 750 }) });']
```

Comment:
instead of `""`, I would suggest `"reading"` since that's what would be used in a real event.